lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -205,7 +211,7 @@ def self.resolve_version(ref, source)
     # @return [true, false]
     #
     def self.sha_hash?(rev)
-      rev =~ /^[0-9a-f]{4,40}$/
+      rev =~ /^[0-9a-fA-F]{4,40}$/
```

Comment:
Let's just add `/i` to the end to make it case insensitve; that's more efficient anyway since it's a smaller scan set and a numeric comparison operation.